src/string_bytes.cc

```diff
@@ -406,9 +407,7 @@ size_t StringBytes::Write(Isolate* isolate,
           reinterpret_cast<uintptr_t>(buf) % sizeof(uint16_t);
       if (is_aligned) {
         uint16_t* const dst = reinterpret_cast<uint16_t*>(buf);
-        for (size_t i = 0; i < nchars; i++)
-          dst[i] = dst[i] << 8 | dst[i] >> 8;
-        break;
+        SwapBytes(dst, dst, nchars);
```

Comment:
It was pointed out in #7645 that this change introduces a bug: the `break` statement should have been retained because now the bytes are swapped back by the fallback path below.